packages/core/src/utils/initializeConfig.ts

```diff
@@ -1,7 +1,12 @@
+import { defaultsDeep } from 'lodash-es';
 import { cloneDeep } from 'lodash-es';
 import { FlatbreadConfig, LoadedFlatbreadConfig, Transformer } from '../types';
 import { toArray } from './arrayUtils';
 
+function camelCase(field: string) {
+  return field.replace(/\s(\w)/g, (_, m) => m.toUpperCase());
+}
```

Comment:
I think this would be valuable as its own exported util. 

For future, we could use [change-case](https://www.npmjs.com/package/change-case) if we end up needing to convert between a handful of cases.

Reply:
Also does this camel case uppercase strings like `TEST field`?

If not, could we could do something like `field.toLowerCase().replace(/\s(\w)/g, (_, m) => m.toUpperCase());`